osbs/core.py

```diff
@@ -872,6 +872,23 @@ def restore_resource(self, resource_type, resource):
         check_response(response)
         return response
 
+    def create_config_map(self, config_data):
+        url = self._build_k8s_url("configmaps/")
+        response = self._post(url, data=json.dumps(config_data))
+        check_response(response)
+        return response
+
+    def get_config_map(self, config_name):
+        url = self._build_k8s_url("configmaps/")
+        response = self._get(url, data=json.dumps(config_name))
+        check_response(response)
+        return response
+
+    def delete_config_map(self, config_name):
+        url = self._build_k8s_url("configmaps/")
+        response = self._delete(url, data=json.dumps(config_name))
```

Comment:
Reading through the rest of the document and comparing, I think 'body' actually means the body of the request (the 'data=' part of the _delete() call), so I suppose it's required in the sense that requests have content.

Yes, `data='{}'` seems fine.